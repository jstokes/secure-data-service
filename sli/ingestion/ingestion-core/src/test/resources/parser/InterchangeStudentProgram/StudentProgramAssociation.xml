<?xml version="1.0" encoding="UTF-8"?>
<InterchangeStudentProgram xmlns:sli="http://slc-sli/ed-org/0.1"
 xmlns="http://ed-fi.org/0100"
 xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
 xsi:schemaLocation="http://ed-fi.org/0100 file:/C:/beta/sli/sli/edfi-schema/src/main/resources/edfiXsd-SLI/SLI-Interchange-StudentProgram.xsd">
    <StudentProgramAssociation id="ID001">
        <StudentReference>
            <StudentIdentity>
                <StudentUniqueStateId>Pt_uyzV4Y0BFCjlg2d</StudentUniqueStateId>
            </StudentIdentity>
        </StudentReference>
        <ProgramReference>
            <ProgramIdentity>
                <ProgramId>OtgtFpQRYbfv__</ProgramId>
            </ProgramIdentity>
        </ProgramReference>
        <Services id="ID003" ref="ID002">
            <ShortDescription>x6So5.qOhsHap-r0JfU80Ap8EHnof</ShortDescription>
        </Services>
        <BeginDate>2014-10-04</BeginDate>
        <EndDate>2013-10-05</EndDate>
        <ReasonExited>Died or is permanently incapacitated</ReasonExited>
        <EducationOrganizationReference>
            <EducationalOrgIdentity>
                <StateOrganizationId>c88eM1JuQv07CD1r4ssi3Jng</StateOrganizationId>
            </EducationalOrgIdentity>
        </EducationOrganizationReference>
    </StudentProgramAssociation>
</InterchangeStudentProgram>
