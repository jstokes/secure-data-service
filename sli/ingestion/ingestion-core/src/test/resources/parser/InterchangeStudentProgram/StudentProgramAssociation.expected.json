{
  "@id":"ID001",
  "StudentReference":{
    "StudentIdentity":{
      "StudentUniqueStateId":{"_value":"Pt_uyzV4Y0BFCjlg2d"}
    }
  },
  "ProgramReference":{
    "ProgramIdentity":{
      "ProgramId":{"_value":"OtgtFpQRYbfv__"}
    }
  },
  "Services":[{
    "@id":"ID003",
    "@ref":"ID002",
    "ShortDescription":{"_value":"x6So5.qOhsHap-r0JfU80Ap8EHnof"}
  }],
  "BeginDate":{"_value":"2014-10-04"},
  "EndDate":{"_value":"2013-10-05"},
  "ReasonExited":{"_value":"Died or is permanently incapacitated"},
  "EducationOrganizationReference":{
    "EducationalOrgIdentity":{
      "StateOrganizationId":{"_value":"c88eM1JuQv07CD1r4ssi3Jng"}
    }
  }
}